tensorflow/compiler/xla/service/gpu/cudnn_fused_conv_rewriter.cc

```diff
@@ -216,6 +216,15 @@ StatusOr<std::unique_ptr<HloInstruction>> TryRewriteToCudnnForwardRelu(
   }
 
   CHECK(bias);
+
+  // Cast integer type bias to float to match 
+  // cudnnConvolutionBiasActivationForward.
+  if (primitive_util::IsIntegralType(bias->shape().element_type())) {
```

Comment:
Should the requirement "biases should be floats" be pushed to the users?

Reply:
I got the impression that we prefer taking less assumption from user and trying to make the transforms comprehensive, when I asked about which front-end APIs inferring integer convolution.